src/comp_loinc/cli.py

```diff
@@ -95,9 +95,8 @@ def _find_loinc_release_path(cls, loinc_dir: Path, work_dir: Path) -> Path:
             raise ValueError(f'LOINC release path {loinc_release_path}/LoincTable is not found.')
         return loinc_release_path
 
-    @classmethod
-    def _find_trees_path(cls, loinc_trees: Path, work_dir: Path) -> Path:
-        loinc_trees_path = None
```

Comment:
Delete superfluous declaration
Just minor edit. My code checker caught this. Removed this leading variable declaration from both funcs. Since there is an if/else, it's impossible for this variable to not be declared.